keyvault/data-plane/pom.xml

```diff
@@ -13,7 +13,7 @@
     <relativePath>../../pom.client.xml</relativePath>
```

Comment:
@JonathanGiles do you know if having a reference a SNAPSHOT version for the parent pom will cause any issues for people consuming this JAR if we publish a non-SNAPSHOT version of these packages?

Reply:
I would rather that we start publishing releases of all 'pom' artifacts as releases, rather than snapshots. I am happy with versioning it as 1.0.0 or 1.0.0-rc1 (or -alpha1 / -beta1, but I think -rc1 is probably best).

Reply:
So your suggesting that we also publish the parent poms at the same time? Would you suggest we do that as on-off or every time we publish a package? If we do it every time that would seem like we would need to version these parent POMs often.

Reply:
I would only publish new pom releases when they have changed. In the short term this will be more frequent than a bit further down the road when we reach more of a steady state.

Reply:
And to clarify, yes, versions will increment each time and often. I am ok with that. I would like to ensure that all child modules kept current with the latest versions also (so when key vault depends on a newer version, we would also update batch, event hubs, etc).